examples/petstore/config.yaml

```diff
@@ -8,7 +8,9 @@ server:
 
 api:
     specs:
-        - path: petstore.yaml
+        - path:
```

Comment:
Let's better not do that or otherwise we should also implement the logic for this. Instead, put `add.yaml` to `tests/test_files` and use it in your unit tests (and leave this file as it was).